### ISSUE TITLE
[Staging] Rounding rules

### ISSUE THREAD

**Issue by szabozoltan69** [IFRCGo/go-api#1272]
## Issue
Some figures on GO frontend appears like this:
![kép](https://user-images.githubusercontent.com/6050643/149520711-14ad302c-9ac9-4c46-ba72-d4286fb67baa.png)

## Expected behaviour
GO should round figures already on backend and send the usable values.
Not only to integers, but:
For numbers between 100 and 1000, we have rounded to the nearest 10.
For numbers greater than or equal to 1000, we have rounded to the nearest 100.

To be cross-checked with @justinginnetti @LukeCaley @tovari

**Comment by LukeCaley** [comment]
Excellent work, thanks @szabozoltan69

**Comment by justinginnetti** [comment]
Thanks, @szabozoltan69 -- when you are ready for me to check this, let me know. Perhaps once DFS has pushed the risk module to the staging server?

**Comment by justinginnetti** [comment]
@LukeCaley and @tovari , are the rounding rules above okay with you? Maybe we see what our figures look like on the staging server with these rules in place and then if something looks off we can adjust?

**Comment by szabozoltan69** [comment]
@justinginnetti Yes, with a clear starting point it's easier to find the root cause. If it was discovered via the risk module, then after that deployment it's better to begin. Or if you can tell an other issue (which is already on the server) then I can begin it with that.

**Comment by nanometrenat** [comment]
Hi @szabozoltan69 is further work (either reqs/design or dev) required on this ticket?
FYI I notice that the total funding requirements sent in the weekly automatic update emails are now into the billions but no drama!
![image](https://github.com/IFRCGo/go-api/assets/33625192/683f8a76-280b-4402-bc7f-b3100b510068)

**Comment by szabozoltan69** [comment]
Thanks @nanometrenat – I think that the rewrote frontend already handles these rules fine.
(Maybe backend/emailsending not so perfectly, true. Could you paste here an example?)

**Comment by nanometrenat** [comment]
Thanks @szabozoltan69 - if the rewritten front end handles this fine then that's the original ticket solved, thank you, I will close this ticket now.

If someone wants to change the emails then they can log a separate ticket about that (with xref).  THANKS!

**Comment by justinginnetti** [comment]
I don't think this issue has been fully resolved. Here's a screenshot from the [Cameroon country page's Risk Watch tab](https://go.ifrc.org/countries/41/risk-watch): 
<img width="1315" alt="Screenshot 2024-04-16 at 13 28 12" src="https://github.com/IFRCGo/go-api/assets/74773708/7523cbb0-b2a3-4e72-aaf6-06d47ea76b08">

**Comment by nanometrenat** [comment]
Thanks @szabozoltan69 - @LukeCaley confirms we're to discuss the business rules around this before confirming 100% how widely (beyond risk pages) the rounding should apply.
